src/uwtools/rocoto.py

```diff
@@ -261,7 +261,9 @@ def _add_task_dependency_strequality(self, e: _Element, config: dict, tag: str)
         :param config: Configuration data for the tag.
         :param tag: Name of new element to add.
         """
-        self._set_attrs(SubElement(e, tag), config)
+        e = SubElement(e, tag)
+        for k, v in config.items():
+            self._add_compound_time_string(e, v, k)
```

Comment:
This is the fix for the `streq` piece. This dependency needs two subelements, which may contain compound time strings.